README.md

```diff
@@ -14,11 +14,11 @@ cd waku-simulator
 Configure the simulation parameters. You can place the env variable in an `.env` file.
 
 ```
-export GOWAKU_IMAGE=statusteam/go-waku:latest
-export NWAKU_IMAGE=statusteam/nim-waku:v0.18.0-rc.0
+export GOWAKU_IMAGE=wakuorg/go-waku:latest
+export NWAKU_IMAGE=wakuorg/nwaku:v0.21.2-rc.0
 export NUM_NWAKU_NODES=5
 export NUM_GOWAKU_NODES=5
```

Comment:
can we set NUM_GOWAKU_NODES=0?
in future Pr we will reasses if we should keep go or not.